### DISCUSSION TITLE
无法站点搜索，日志报错

### DISCUSSION THREAD

**Discussion by MapleRedbean** [jxxghp/MoviePilot#2605] [问题咨询] [answered]
【WARNING】2024-07-25 00:02:09,472 - search.py - 霸王别姬 未搜索到资源
【INFO】2024-07-25 00:02:09,471 - search.py - 站点搜索完成，有效资源数：0，总耗时 1 秒
【INFO】2024-07-25 00:02:09,469 - search.py - 站点搜索进度：1 / 1
【WARNING】2024-07-25 00:02:09,467 - indexer - 馒头 未搜索到数据，耗时 1 秒
**【ERROR】2024-07-25 00:02:09,350 - indexer - 馒头 搜索出错：'NoneType' object has no attribute 'get'**
【INFO】2024-07-25 00:02:07,607 - search.py - 开始搜索资源，关键词：霸王别姬 ...

这是什么问题呢

**Comment by MapleRedbean** [comment]
站点添加成功，连通性绿色

**Reply by BH2TYQ** [reply, marked as answer]
你好，请教一下馒头的站点配置方法

**Reply by shaopyu** [reply]
我现在也是这个问题，上个月的时候还是好的，你现在问题解决了吗？

**Reply by MapleRedbean** [reply]
令牌添加成密匙了 你再试试

**Reply by shaopyu** [reply]
感谢，现在解决了：）

**Reply by XiaoHuoZZ** [reply]
> 感谢，现在解决了：）

@shaopyu 你好，想请问是怎么解决的呢，我也遇到这个问题了，连通以及手动搜索都ok，就订阅不行

**Reply by shaopyu** [reply]
> > 感谢，现在解决了：）
> 
> @shaopyu 你好，想请问是怎么解决的呢，我也遇到这个问题了，连通以及手动搜索都ok，就订阅不行

过去时间有点久了，具体记不清了。应该是之前我设置令牌的时候错误的设置成密匙，实际应该是“控制台-实验室-存取令牌”。
